docker-compose.yml

```diff
@@ -17,6 +17,8 @@ services:
         volumes:
             - .:/var/www/html:cached
             - ./src/Frontend/Files:/var/www/html/src/Frontend/Files:cached
+            - fork-cms-cache:/var/www/html/var/cache
+            - fork-cms-logs:/var/www/html/var/logs
```

Comment:
Hey @jessedobbelaere ! Why do you create these volumes? Won't they make the application slower?

Reply:
Hmm, the reason why I added it now (and not earlier), is because there were errors for the logs and cache dir in the Travis job I added when Fork CMS starts up: 

```
RuntimeException: Unable to write in the "cache" directory (/var/www/html/var/cache/install)
```

At the company I work for, we also keep Symfony's cache and logs in an internal volume. I wouldn't say it slows performance down, it should actually improve performance because it will not continuously sync the big cache and logs dir from inside Docker to the host machine (your mac/windows). For that reason, [we also usually keep the `vendor/` folder in its own volume](http://blog.michaelperrin.fr/2017/04/14/docker-for-mac-on-a-symfony-app/#solution1movevendorsoutoftheshareddirectory), so Docker does not have to sync between Mac and the container all the time. But I didn't add the vendor folder here because it causes confusion for people: they run composer install on their mac and it does not reflect in the container because there's no mount of that folder... 

The `logs/` and `cache/` directories stay in their own internal volume, which means it does not mount the directory from your mac too which gave the permission problems (you would need to `chown` the directories again like the Dockerfile does). The only downside is that people need to clear the logs/cache in their container (which they normally do via the `bin/console c:c` command) instead of deleting files on their mac. Or you can do `docker-compose down -v` to drop the project's volumes and start fresh again.

Reply:
Then maybe we should mention this cache stuff in the README file.

Reply:
@jessedobbelaere can you still add that to the docs in this PR?
Maybe also show the exact command they need to run to clear it in the docker container

Reply:
Yeah the docs about Docker can improve 👍 will take a look at this this evening

Reply:
Pushed some changes to the docs 👍

Reply:
I also tried out [the new Mutagen-based caching](https://github.com/docker/for-mac/issues/2707#issuecomment-631606521) in the latest Docker for Mac (Edge!) which speeds up things a lot, so I could get rid of my Laravel Valet/Symfony server setup. 

One of the things mentioned on the [Mutagen-based caching page](https://docs.docker.com/docker-for-mac/mutagen-caching/#avoiding-synchronizing-a-subdirectory) is:

> **Avoiding synchronizing a subdirectory**
> Although two-way file sync is suitable for many types of files, sometimes containers can generate lots of data which doesn’t require copying to the host, for example, debug logs. If your project has a subdirectory that doesn’t need to be continuously copied back to the host, then use a named docker volume to bypass the sync.

Which now works well because the `var` folder (logs/cache) is already "ignored" by Mutagen caching because it's on a named volume 👍